config/export_kc_config.sh

```diff
@@ -29,21 +29,23 @@ fi
 # Define a temporary file to store the modified realm.json
 MODIFIED_REALM_JSON="modified_realm.json"
 
-# Replace the placeholder 'KEYCLOAK_KEYSTORE_PATH' in the realm.json file with the actual value from the .env
+# Replace the placeholders 'KEYCLOAK_KEYSTORE_PATH','KEYCLOAK_KEYSTORE_PASSWORD' and 'CLIENT_SECRETin' in the realm.json file with the actual value from the .env
 sed -e "s|KC_KEYSTORE_PATH|$KC_KEYSTORE_PATH|g" \
     -e "s|KEYCLOAK_KEYSTORE_PASSWORD|$KEYCLOAK_KEYSTORE_PASSWORD|g" \
     -e "s|CLIENT_SECRET|$CLIENT_SECRET|g" \
-    $IMPORT_PATH > $MODIFIED_REALM_JSON
-
+    $KC_REALM_FILE > $MODIFIED_REALM_JSON
 
 # Run the JAR file with the specified parameters
 echo "Running the JAR file..."
-java -jar target/$JAR_FILE \
+java -jar target/$KC_CLI_JAR_FILE \
   -Dimport-realm="true" \
   -Dforce="true" \
   --keycloak.url="$KEYCLOAK_URL" \
-  --keycloak.user="$KEYCLOAK_USER" \
-  --keycloak.password="$KEYCLOAK_PASSWORD" \
+  --keycloak.user="$KEYCLOAK_ADMIN" \
+  --keycloak.password="$KEYCLOAK_ADMIN_PASSWORD" \
   --keycloak.ssl-verify="true" \
   --import.files.locations="$MODIFIED_REALM_JSON" || { echo "Failed to run the JAR file"; exit 1; }
-echo "Script completed successfully."
\ No newline at end of file
+echo "Script completed successfully."
+
+#Removing MODIFIED_REALM_JSON form memory
+unset MODIFIED_REALM_JSON
```

Comment:
You're creating a file on Line 36. I meant deleting the file, not de-initializing the variable with the name of the file.